src/Main.java

```diff
@@ -1,5 +1,114 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Типы переменных 2");
+
+        // Задача 1
+
+        int semechek = 1000000;
+        byte car = 60;
+        short derevo = 300;
+        long galaktik = 3000000;
+        float people = 6.314567f;
+        double rastenij = 9.456852198d;
+        System.out.println("Значение переменной int semechek с типом int равно " + semechek);
+        System.out.println("Значение переменной byte car с типом byte равно " + car);
+        System.out.println("Значение переменной short derevo с типом short равно " + derevo);
+        System.out.println("Значение переменной long galaktik с типом long равно " + galaktik);
+        System.out.println("Значение переменной float people с типом float равно " + people);
+        System.out.println("Значение переменной double rastenij с типом double равно " + rastenij);
+
+        // Задача 2
+
+        float a = 27.2f;
+        long b = 987678965549l;
+        double c = 2.786;
+        int d = 569;
+        short e = -159;
+        short f = 27897;
+        byte g = 67;
+
+        // Задача 3
+
+        short lu = 23;
+        short an = 27;
+        short ek = 30;
+        short listov = 480;
+        int na1 = 480 / (lu + an + ek);
+        System.out.println("На каждого ученика рассчитанно " + na1 + " листов бумаги ");
+
+        // Задача 4
+
+        int but = 16;
+        int time = 2;
+        int time1 = 20;
+        int sutki = 1;
+        int den3 = 3;
+        int mesac = 1;
+        int min20 = but / time * time1;
+        int sutki1 = 8 * 60 * 24;
```

Comment:
```suggestion
        int sutki1 = but / time * 60 * 24;
```
надо использовать переменные, а не числа